config/storage.yml

```diff
@@ -6,13 +6,17 @@ local:
   service: Disk
   root: <%= Rails.root.join("storage") %>
 
-# Use rails credentials:edit to set the AWS secrets (as aws:access_key_id|secret_access_key)
-# amazon:
-#   service: S3
-#   access_key_id: <%= Rails.application.credentials.dig(:aws, :access_key_id) %>
-#   secret_access_key: <%= Rails.application.credentials.dig(:aws, :secret_access_key) %>
-#   region: us-east-1
-#   bucket: your_own_bucket
+# Here we set the S3 bucket to the appropriate style, unless we override it in the ENV variables.
+<% s3_bucket = Figaro.env.AWS_S3_BUCKET.nil? ? "caucus-#{(Hash["production", "prod", "development", "dev", "test", "ci"])[Rails.env]}" : Figaro.env.AWS_S3_BUCKET %>
```

Comment:
Agreed, but I couldn't figure out where exactly this helper would exist, since it's way before the main stack is initialized. Im going to leave it for the moment.